src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -584,10 +584,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // Windows 10.
         // We'll also turn the acrylic back off when they're fully opaque, which
         // is what the Terminal did prior to 1.12.
-        if (!IsVintageOpacityAvailable())
-        {
-            _runtimeUseAcrylic = newOpacity < 1.0;
-        }
+        UseAcrylic(newOpacity < 1.0 && (!IsVintageOpacityAvailable() || _settings->UseAcrylic()));

```

Comment:
Worried about this one instance of calling `UseAcrylic()` where below we directly-set `_runtimeUseAcrylic`. Should we decide on one or the other?

Reply:
Also, we will want to change the implementation of `_isBackgroundTransparent` to account for this, right?

Reply:
Unless, changing this here feeds into `_isBackgroundTransparent` correctly. But then, why does `_isBackgroundTransparent` check `Opacity` on its own? I'm confused.

Reply:
> Unless, changing this here feeds into `_isBackgroundTransparent` correctly.

Yes, this does seem to be the case because `_runtimeUseAcrylic` is updated (with the new constraints) before `_isBackgroundTransparent` is called on [L715](https://github.com/microsoft/terminal/blob/a0b526ef112fc2e1f5dfda0a432d1b1f8f62bcc4/src/cascadia/TerminalControl/ControlCore.cpp#L715). This was not something that I took into account when I created this PR, and I fully agree that it should be modified to take the new constraints into account. This amounts to removing `UseAcrylic` from the expression.

> But then, why does `_isBackgroundTransparent` check `Opacity` on its own? I'm confused.

I suppose the `Opactity` was checked on its own to account for the regular transparent background on win11.